src/debuggers/natvis/magnum.natvis

```diff
@@ -38,16 +39,64 @@
     <DisplayString IncludeView="simple">{{ {_data[0],g}, {_data[1],g} }}</DisplayString>
     <DisplayString ExcludeView="simple">{{ x={_data[0],g} y={_data[1],g} }}</DisplayString>
   </Type>
+  <!-- Math::Vector2< Float > -->
+  <Type Name="Magnum::Math::Vector&lt;2,float&gt;">
+    <Intrinsic Name="Dot" Expression="_data[0]*_data[0]+_data[1]*_data[1]"></Intrinsic>
+    <Intrinsic Name="IsNormalized" Expression="Dot()&gt;0.99980000000500002&amp;Dot()&lt;1.0001999999950000"></Intrinsic>
```

Comment:
It might be worth to add a comment with how the values were calculated in case the epsilons in TypeTraits get updated in the future. Something like `<!-- 2*epsilon + epsilon*epsilon, epsilon = 1.0e-whatever  -->` could be sufficient I think, similarly for the double version.

I'm thinking if it would be better to use the full expression in there, but since the same value is repeated several times, it'd be a lot of noise. Not sure.

Reply:
I can add the comments right away.

Regarding writing the full expressions, I'd say it would be clearer from a maintenance perspective. Let me know if you'd like me to revise it.

Reply:
> I'd say it would be clearer from a maintenance perspective.

Yeah, true, then do that :)

Reply:
Oh wait I see now, yeah that line would get really long and harder to read if we put the epsilon's literal value and add/subtract with 1.0. Plus a lot of repetition of this same line.

How about we place a single comment block up top and explain how the values were calculated?

Reply:
Yes so basically what I suggested at first :)

Reply:
How is this for the comment?

```
  ...

<!--For Vector & Quaternion types, IsNormalized() functionality was implemented by comparing the Dot() against 1.0f/1.0 with 2*epsilon
    where epsilon = 1.0e-5f for float & epsilon = 1.0e-14 for double variants.-->
  
  <!-- Math::Vector2 -->
  <Type Name="Magnum::Math::Vector&lt;2,*&gt;">
    <DisplayString IncludeView="simple">{{ {_data[0],g}, {_data[1],g} }}</DisplayString>
    <DisplayString ExcludeView="simple">{{ x={_data[0],g} y={_data[1],g} }}</DisplayString>
  </Type>
  
  ...
```